src/main/java/com/google/api/codegen/transformer/ApiMethodTransformer.java

```diff
@@ -238,6 +240,12 @@ public StaticLangApiMethodView generateCallableMethod(MethodTransformerContext c
         !ServiceMessages.s_isEmptyType(context.getMethod().getOutputType()));
     methodViewBuilder.isPageStreaming(false);
 
+    MethodConfig methodConfig = context.getMethodConfig();
+    if (methodConfig.isGrpcStreaming()) {
+      GrpcStreamingConfig grpcStreaming = methodConfig.getGrpcStreaming();
+      methodViewBuilder.streamingType(grpcStreaming.getType());
+    }
```

Comment:
Maybe it would be easier to have this if condition elsewhere, so that we could just always set streamingType (and set it to None if there is no streaming)